packages/react/src/components/Table/TableHead/FilterHeaderRow/_filter-header-row.scss

```diff
@@ -15,9 +16,56 @@
   }
 }
 
+.#{$prefix}--data-table
+  th.#{$iot-prefix}--filter-header-row--header.#{$iot-prefix}--tableheader-filter {
+  padding-right: $spacing-02;
+  padding-left: $spacing-02;
+  padding-bottom: $spacing-03;
+
+  &.#{$iot-prefix}--tableheader-filter:not(:first-of-type) {
+    border-left-width: 2px;
```

Comment:
Should these be abstracted into a variable since we use it in a few places?

Reply:
added a variable.